package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@savingsunited/zvezdochki",
-  "version": "1.2.0",
+  "version": "1.3.0",
```

Comment:
Since we removed some of the features, let's target `2.0.0` (breaking change).

Reply:
Got it!